src/components/Filter/FilterHelper.spec.tsx

```diff
@@ -0,0 +1,468 @@
+      code: FilterType.Claim,
+      filterOptions: [
+        {
+          key: 'm1',
+          label: 'Problem Solving',
+          isSelected: true,
+          filterType: FilterType.Claim
+        },
+        {
+          key: 'm2',
+          label: 'Other math stuff',
+          isSelected: false,
+          filterType: FilterType.Claim
+        }
+      ]
+    };
+    expect(result.claimFilter).toEqual(expectation);
+  });
+
+  it('target none selected', () => {
+    const params = { grades: [], subject: 'math', claim: 'm1' };
+    const result = createFilters(filterOptions, params);
+
+    const expectation: AdvancedFilterCategoryModel = {
+      isMultiSelect: false,
+      displayAllButton: true,
+      disabled: false,
+      label: 'Target',
+      code: FilterType.Target,
+      filterOptions: [
+        {
+          key: 'sr',
+          label: 'Short Response',
+          isSelected: false,
+          filterType: FilterType.Target
+        },
+        {
+          key: 'pf',
+          label: 'Proof',
+          isSelected: false,
+          filterType: FilterType.Target
+        }
+      ]
+    };
+    expect(result.targetFilter).toEqual(expectation);
+  });
+
+  it('target pf selected', () => {
+    const params = { grades: [], subject: 'math', claim: 'm1', target: 'pf' };
+    const result = createFilters(filterOptions, params);
+
+    const expectation: AdvancedFilterCategoryModel = {
+      isMultiSelect: false,
+      displayAllButton: true,
+      disabled: false,
+      label: 'Target',
+      code: FilterType.Target,
+      filterOptions: [
+        {
+          key: 'sr',
+          label: 'Short Response',
+          isSelected: false,
+          filterType: FilterType.Target
+        },
+        {
+          key: 'pf',
+          label: 'Proof',
+          isSelected: true,
+          filterType: FilterType.Target
+        }
+      ]
+    };
+    expect(result.targetFilter).toEqual(expectation);
+  });
+
+  it('disables claim for no subject', () => {
+    const params = { grades: [] };
+    const result = createFilters(filterOptions, params);
+
+    expect(result.claimFilter.disabled).toEqual(true);
+  });
+
+  it('disables target for no subject', () => {
+    const params = { grades: [] };
+    const result = createFilters(filterOptions, params);
+
+    expect(result.targetFilter.disabled).toEqual(true);
+  });
+
+  it('disables target for no claim', () => {
+    const params = { grades: [], subject: 'math' };
+    const result = createFilters(filterOptions, params);
+
+    expect(result.targetFilter.disabled).toEqual(true);
+  });
+});
+
+describe('FilterHelper.sanitizeParams', () => {
+  it('does nothing with empty params', () => {
+    const params = { grades: [] };
+    const result = sanitizeParams(params, filterOptions);
+
+    expect(result).toEqual(params);
+  });
+
+  it('does nothing on correct case', () => {
+    const params = { grades: ['ms', 'hs'], subject: 'math', claim: 'm1', target: 'pf' };
```

Comment:
Same here, would you extract these out into a `beforeEach` or `beforeAll` call and change the properties in each test as needed.